src/js/users/classes.ts

```diff
@@ -0,0 +1,65 @@
+import { faker } from "@faker-js/faker";
+import { times } from "lodash-es";
+
+type Group = {
+    id: string;
+    name: string;
+    toggled: boolean;
+};
+
+type Permissions = {
+    cancel_job: boolean;
+    create_ref: boolean;
+    create_sample: boolean;
+    modify_hmm: boolean;
+    modify_subtraction: boolean;
+    remove_file: boolean;
+    remove_job: boolean;
+    upload_file: boolean;
+};
+
+class UserDetail {
+    active: boolean;
+    administrator: boolean;
+    force_reset: boolean;
+    groups: Group[];
+    handle: string;
+    id: string;
+    last_password_change: string;
+    permissions: Permissions;
+    primary_group: Group;
+}
+
+export function getFakeUserDetail(): UserDetail {
+    const groups = [];
+    times(5, (i: number) =>
+        groups.push({ id: faker.random.alphaNumeric(9, { casing: "lower" }), name: `group${i}`, toggled: true })
+    );
```

Comment:
Based on the Lodash docs, this should work.
```suggestion
    const groups = times(5, (i: number) => ({
        id: faker.random.alphaNumeric(9, { casing: "lower" }),
        name: `group${i}`,
        toggled: true
    }));
```